.github/workflows/ci.yml

```diff
@@ -96,19 +105,40 @@ jobs:
           - gcc-7
           - gcc-8
     steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
I'm pretty sure this is unnecessary. The only tests right now that do checkout are ones that don't use builder

(huh, looks like the new `openbsd` test does checkout ... probably unnecessary)

Reply:
If memory serves, builder tries to clone the project repo before it does the installation of platform packages. This doesn't work on OpenBSD because git isn't available in the base OS install and has to be installed from packages.